SwiftWisdomTests/StandardLibrary/Date/NSDateComparableTests.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  NSDateComparableTests.swift
+//  SwiftWisdom
+//
+//  Created by Patrick Butkiewicz on 3/1/17.
+//  Copyright © 2017 Intrepid. All rights reserved.
+//
+
+import Foundation
+import XCTest
+
+final class NSDateComparableTests: XCTestCase {
+    let olderDate = NSDate(timeIntervalSince1970: 0)
+    let olderDateCopy = NSDate(timeIntervalSince1970: 0)
+    let moreRecentDate = NSDate(timeIntervalSinceNow: 0)
+    
+    func testLessThanDate() {
+        XCTAssert(olderDate < moreRecentDate)
```

Comment:
xctassertfalse(olderDate < olderDateCopy) ..  dont exactly think this should fail but...